samples/Makefile

```diff
@@ -471,9 +489,10 @@ zip:
 # Clean-up rules
 
 mostlyclean:
-	@$(DEL) *.lbl *.map *.o *.s 2>$(NULLDEV)
+	@$(DEL) *.lbl *.map *.bin *.hex *.o *.s 2>$(NULLDEV)
```

Comment:
- The ld65 binary output files have no special file extension, so there's no need to remove .bin files.
- There's no cc65 program generating .hex files so there's no need to remove them. 

So please remove this change again.

Reply:
**DONE**.  Please check and confirm.